Cognigy/chat-components
# Feature/adaptive card heading a11y

This PR adds aria-level attribute to adaptive card headings. It also modifies the caption label on videos with captions.

Success criteria:
- When the 'style: heading' is added to a text block in adaptive card, the aria-level=2 attribute should be added to the heading element
- The video with caption should have the label 'On' in the menu overlay, instead of the label 'English'
- 
How to test:
- In the demo chat, inspect the adaptive card header. Check if the aria-level=2 attribute is present with the role=heading attribute 
- In the demo, turn on the captions in the video player. Open the secondary menu again and check the label below 'Captions' menu item. It should say 'On' instead of 'English'
